pyflakes/checker.py

```diff
@@ -663,17 +664,27 @@ def getNodeName(node):
         return node.name
 
 
+class ANY_TYPING_NAME(object):
+    """
+    Marker for asking if the given node is a valid but unspeciffied typing member.
+
+    Note: this is *not* asking if the member is `typing.Any`.
+    """
+
+
 def _is_typing(node, typing_attr, scope_stack):
+    typing_modules = {'typing', 'typing_extensions'}
```

Comment:
this would be slightly better as a module constant (so it doesn't rebuild the set on every call)

Reply:
Done.

Fun fact: I've found that for small collections, comparing to membership of a `tuple` can sometimes actually be faster than sets, especially if the collection is declared inline. (Testing this case doesn't show a meaningful difference, so I've left it as a set.)

Reply:
yep -- a tuple can be inlined into the code object (the peephole optimizer also does this for `in [1, 2, 3]` and `in {1, 2, 3}` as well -- converting the former to a tuple and the latter to a frozenset)

```pycon
>>> def f(x): return x in (1, 2, 3)
... 
>>> def g(x): return x in [1, 2, 3]
... 
>>> def h(x): return x in {1, 2, 3}
... 
>>> import dis
>>> dis.dis(f)
  1           0 LOAD_FAST                0 (x)
              2 LOAD_CONST               1 ((1, 2, 3))
              4 CONTAINS_OP              0
              6 RETURN_VALUE
>>> dis.dis(g)
  1           0 LOAD_FAST                0 (x)
              2 LOAD_CONST               1 ((1, 2, 3))
              4 CONTAINS_OP              0
              6 RETURN_VALUE
>>> dis.dis(h)
  1           0 LOAD_FAST                0 (x)
              2 LOAD_CONST               1 (frozenset({1, 2, 3}))
              4 CONTAINS_OP              0
              6 RETURN_VALUE
```